src/main/resources/MessagingDescriptor.json

```diff
@@ -5,8 +5,14 @@
       "description": "Patron fee/fine balance was changed",
       "eventTTL": 1,
       "signed": false
+    },
+    {
+      "eventType": "FEEFINE_WITH_LOAN_CLOSED",
```

Comment:
Does this event refer to a fee / fine that has a loan being closed, or something to do with a fee / fine with a closed loan?

Reply:
this event refers to a fee/fine that was recently closed and has a loan associated.

Reply:
> this event refers to a fee/fine that was recently closed and has a loan associated.

Why is it important to have separate events for fees / fines that are closed with a link to a loan and fees / fines that are closed without a link to a loan?

Can it be generalised to `fee / fine closed` and optionally including the linked loan ID in the event representation?

Reply:
This will increase load on mod-circulation. Each time any account is closed pub/sub will notify mod-circulation about this. 

An account _without_ a loan it is a manual charge and mod-circulation is not interesting in such events. 

Should I proceed anyway?

Reply:
> This will increase load on mod-circulation. Each time any account is closed pub/sub will notify mod-circulation about this.
> 

That's true, there could be additional load. Can circulation complete the request quickly if there is a no loan ID?

> An account _without_ a loan it is a manual charge and mod-circulation is not interesting in such events.

Are some manual charges related to loans e.g. actual cost fees?

> Should I proceed anyway?

It might be that we want a specialised event. I'd prefer to wait to decide that, unless we can think of a really explicit name like `LOAN_RELATED_FEE_FINE_CLOSED`?

Reply:
> Can circulation complete the request quickly if there is a no loan ID?

Sure, it can.

Reply:
> Are some manual charges related to loans e.g. actual cost fees?

Yes, actual cost fees are going to be manual charges, but I'd be good to have a link to the loan for them as well.

Reply:
> This may result into a situation when the event payload will be constantly updated with some extra-fields, that needed for a particular subscriber, which may lead to another one dependency on the payload version.
> 
> I'd prefer to have specialized event rather then an generalized event with extra-properties in payload.

Ok, I'm happy to go with that, as long as we can make the event name very specific and unambiguous.

Reply:
Ok, I'll proceed with `LOAN_RELATED_FEE_FINE_CLOSED` name then?

Reply:
> Ok, I'll proceed with `LOAN_RELATED_FEE_FINE_CLOSED` name then?

I'm ok with that. I'm happy for you to think of a better name. The most important aspect is that it is unambiguous about what is closed and how the loan is involved.